package.json

```diff
@@ -1,34 +1,8 @@
 {
-  "scripts": {
-    "clean": "rm -rf lib lib-es6",
-    "test": "jest",
-    "test:cov": "jest --config jest.config.ts.js --coverage",
-    "test:watch": "jest --config jest.config.ts.js --watch",
-    "prepare": "yarn clean && tsc && tsc --build tsconfig.es6.json",
-    "benchmark": "matcha benchmark.js"
-  },
-  "devDependencies": {
-    "@types/jest": "^23.3.1",
-    "@types/lodash": "^4.14.116",
-    "@types/node": "^10.7.1",
-    "jest": "^23.5.0",
-    "lodash": "^4.17.10",
-    "matcha": "^0.7.0",
-    "prettier": "^1.14.2",
-    "reselect": "^3.0.1",
-    "ts-jest": "^23.1.4",
-    "typescript": "^3.0.1"
-  },
-  "name": "@taskworld.com/rereselect",
+  "name": "@taskworld/rereselect",
   "description": "A library that generates memoized selectors and supports dynamic dependency tracking.",
-  "version": "0.3.0",
-  "main": "lib/index.js",
-  "directories": {
-    "example": "examples",
-    "lib": "lib"
-  },
-  "dependencies": {},
-  "author": "Thai Pangsakulyanont <thai.p@taskworld.com> (http://dt.in.th/)",
+  "version": "1.0.0",
```

Comment:
Once this PR is merged, I will publish the package onto NPM manually.
I'm too lazy to config a CI + NPM token. Plus, I don't expect we update this library often.